components/organisms/footer.html

```diff
@@ -0,0 +1,97 @@
+<div class=footer>
+
+  <footer class="main-footer">
+
+    <div class="footer-button">
+      <h4>Example</h4>
+      <button type="button">Click here!</button>
+    </div>
+
+    <div class="social">
+      <h4>Connect with us!</h4>
+      <ul class="social-icons">
+        <li><a href="http://www.facebook.com"><img src='../../img/icon_facebook.png' /></a></li>
+        <li><a href="http://www.twitter.com"><img src='../../img/icon_instagram.png' /></a></li>
+        <li><a href="http://www.youtube.com"><img src='../../img/icon_twitter.png' /></a></li>
+      </ul>
+    </div>
+
+  </footer>
+
+</div>
+<style>
+
+  /* =================================
+  Base Layout Styles
+==================================== */
+  .footer {
+    padding-top: .75em;
+    background: rgba(232, 236, 241, 1);
+  }
+  .main-footer {
+    grid-template-columns: 1fr;
+    grid-template-rows: auto;
+    grid-gap: 74px;
+    text-align: center;
+    padding: 2em 0;
+    background: rgba(54, 56, 57, .92);
+    text-align: left;
+    color: white;
+    font-family: arial;
+  }
+
+  .footer-button {
```

Comment:
I'm surprised this class is used on a `<div>` instead of on a `<button>` as the name alludes to. What do you think about creating a new class for the wrapping div and using this class to style the button?

Reply:
I am trying to give margin to button and h4. Will this create problems later? or what is your point here?

Reply:
I mean that naming a class `.footer-button` leads me to believe that this class styles buttons in the footer. However, in this case, the class is only styling the div that surrounds the button, but not actually the button itself.

Do you think it makes sense to create a reusable class for the footer columns? Something like, `.footer-column` which could be applied this `<div>` as well as the social div. Remember that elements can have multiple classes which allows you to build up styles from multiple pieces.

Reply:
Got it! Thank you so much for this review. please review my changes whenever you get a chance.